src/App/Pages/Vault/GroupingsPage/GroupingsPage.xaml.cs

```diff
@@ -64,6 +66,11 @@ public partial class GroupingsPage : BaseContentPage
                 ToolbarItems.Add(_lockItem);
                 ToolbarItems.Add(_exitItem);
             }
+            if (deleted)
+            {
+                _absLayout.Children.Remove(_fab);
+                ToolbarItems.Remove(_addItem);
+            }
```

Comment:
I am trying to recall why we need a case for `deleted` on this page. Isn't this the main listing page where folders and collections are listed? I don't see us having a `deleted` parameter on the kslib implemention of groupings.

Reply:
Acknowledged, I missed this nuance, good catch, updating it now.

Reply:
As discussed, the CiphersPage in mobile behaves a little differently than the JS clients and is primarily used for auto-fill and search, so the Trash bin is using the convention that the other type and collection filters are using by recursing back to the groupings page with a flag set.